.gitignore

```diff
@@ -7,6 +7,7 @@ bundles/io/org.openhab.io.multimedia.tts.marytts/lib/
 .metadata/
 */plugin.xml_gen
 targetplatform/*.launch
+target/*
```

Comment:
Maybe I'm doing something, but that folder is created by using the static code analysis tool. I used it and just  wanted to exclude this folder because it shouldn't be uploaded.